tools/cli/src/errors.rs

```diff
@@ -45,4 +45,10 @@ pub enum CLIError {
     /// Various errors related to I/O operations.
     #[error("{0:?}")]
     IOError(#[from] std::io::Error),
+
+    #[error("{0}")]
+    ManifestParseError(String),
+
+    #[error("{0:?}")]
```

Comment:
Could you use `transparent` instead?

Reply:
I changed all "{0}" on other errors to `transparent`. Ones on `String` variants and `{0:?}` I kept as is, because with `String` it doesn't compile and i think that call to `Debug` instead of `Display` for `std::io::Error` had some meaning

Reply:
Ok, thanks